bot/cogs/todo_list.py

```diff
@@ -0,0 +1,161 @@
+
+                    with open(f"{BASE_DIR}/save_data_td.JSON", "r+") as read:
+                        file = json.load(read)
+                        file.update([(str(ctx.author.id), self.the_todo)])
+
+                        with open(f"{BASE_DIR}/save_data_td.JSON", "w+") as save:
+                            json.dump(file, save)
+
+                            completion_message = f"{ctx.author.mention}, your todo list:\n{''.join(self.the_todo) + '-> Ready!'}"
+                            await ctx.send(completion_message)
+                else:
+                    completion_message = (
+                        f"{ctx.author.mention}, your todo list entry is empty. "
+                        f"Please enter a todo list format (i.e `.todolist Make assignments, Learn Python`) with comma "
+                        f"on "
+                        f"every todo."
+                    )
+                    await ctx.send(completion_message)
+
+                self.todos[ctx.author.id] -= 1
+            elif task_type == "del_todos":
+                if some_todos:
+                    r = some_todos.split(",")
+                    for td in r:
+                        if td == "":
+                            continue
+                        else:
+                            self.del_the_todo.append(f"{td.strip(' ').rstrip(' ')}\n")
+
+                    with open(f"{BASE_DIR}/save_data_td.JSON", "r+") as read:
+                        f = json.load(read)
+                        new = [
+                            x
+                            for x in f[str(ctx.author.id)]
+                            if x not in self.del_the_todo
+                        ]
+                        removed = [
+                            x for x in f[str(ctx.author.id)] if x in self.del_the_todo
+                        ]
+                        f[str(ctx.author.id)] = new
+                        with open(f"{BASE_DIR}/save_data_td.JSON", "w+") as save:
+                            json.dump(f, save)
+                            completion_message = (
+                                f"{ctx.author.mention}, your todo list after deletion:\n{''.join(new)}\n"
+                                f"Removed todos:\n{''.join(removed)}"
+                            )
+                            await ctx.send(completion_message)
+                else:
+                    completion_message = (
+                        f"{ctx.author.mention}, your todo list entry is empty. "
+                        f"Please enter a todo list format (i.e `.del_todos Make assignments, Learn Python`) with comma "
+                        f"on "
+                        f"every todo."
+                    )
+                    await ctx.send(completion_message)
+
+        create_todo_list.start()
+
+    @command(brief="A simple todolist in text")
+    async def todolist(self, ctx, *, some_todos=None):
```

Comment:
command needs a "name," probably `name="todo"`, there is no documentation for how to use this command, needs a description as well.